neon_mq_connector/connector.py

```diff
@@ -247,9 +247,10 @@ def create_mq_connection(self, vhost: str = '/', **kwargs):
         return pika.BlockingConnection(parameters=self.get_connection_params(vhost, **kwargs))
 
     def register_consumer(self, name: str, vhost: str, queue: str,
```

Comment:
Add some unit tests that look like calls to this method that exist currently so we can test for similar breakage in the future